build.gradle.kts

```diff
@@ -16,4 +19,11 @@ project(":samples") {
     }
 }
 
+tasks.named<Task>("check") {
+    dependsOn(gradle.includedBuild("simple-jlink").task(":check"))
+    dependsOn(findProject(":samples:javafx-app")!!.tasks.getByName("jlinkGenerateReleaseImage"))
+    dependsOn(findProject(":samples:simple-jar")!!.tasks.getByName("jlinkGenerateReleaseImage"))
+    dependsOn(findProject(":samples:groovy-dsl")!!.tasks.getByName("jlinkGenerateReleaseImage"))
```

Comment:
Make each sub-project's `check` task depend upon that project's `check` task, not the root `check` task.

Reply:
I'd prefer the samples to be normal projects that use the plugin.

Reply:
These goals aren't contradictory?

Reply:
Well, the way I see it is that the root project just ties the plugin project and the sample projects together. So having the root project's `check` task make sure the samples play well with the plugin makes more sense to me than having each sample project's standard `check` task not only check the normal project things (tests, static analysis, etc) but also check the generated jlink distributions